config/local_dev.env

```diff
@@ -1,5 +1,7 @@
 # Environment variables for the local development environment
 
+ENV=nonprod
```

Comment:
Should it be `local`?

I guess it would imply having a `config.local.yaml`. But it would be better than having the `devtest` action used in unit tests in `config.nonprod.yaml`

Reply:
Alternatively, we opt for one config and make the chosen environments a configuration parameter.

Then the local/nonprod/prod actions would be in a consolidated yaml but enabled/disabled through whichever parameter. (Boolean flag per environment, list of all environments, etc?)